### DISCUSSION TITLE
Cannot find FirebaseFirestore namespace

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#911] [Support / Q&A] [answered]
Is there a way to use the Firestore namespace or any other namespace, say, to use other npm packages? 

For example, Firestore it is available while coding a task. Variables can be typed and object documentation is shown.
![image](https://user-images.githubusercontent.com/94030557/199324726-7b649b4b-c4d1-4695-acf9-b511817d27f2.png)

 But when the function is built, an error stating that the namespace is not available shows up. What am I missing?
![image](https://user-images.githubusercontent.com/94030557/199324230-c756a2d0-f389-4f2f-84c2-40e131006cee.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hi @st-trade, if this is still an issue, perhaps you can join our upcoming [office hours](http://rowy.io/events) or book a [short 1:1 meeting](https://rowy.io/meet) so that we can go over this and get it resolved.
